### PR TITLE
Fix uint256 loading from protobuf

### PR DESCRIPTION
**Pull request by hewigovens**
Related issues:

- #75 
- https://github.com/TrustWallet/private-trust-wallet-ios/issues/868
- https://github.com/TrustWallet/private-trust-wallet-ios/issues/864
- https://github.com/TrustWallet/private-trust-wallet-ios/issues/863

Need to fix `EthereumTests.testSigner`. The problem is that C++ code deserializes wrong protobuf model value from Swift protobuf model, more specifically, wrong BigInt values

```swift
/// gasLimit in Swift
BigInt(21000).magnitude.serialize()
/// becomes "R\b" when reading from C++ side, which is 10504
```

### CODE REVIEW

**Comment by hewigovens** [comment]
probably https://github.com/TrustWallet/private-trust-wallet-ios/pull/867

File: tests/TWEthereumSignerTests.cpp [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+TEST(TWEthereumSigner, BigInt) {
+    /// From Swift BigInt(21000).magnitude.serialize()
+    Data bytes = {0x52, 0x08};
+    auto limit = uint256_t(21000);
+    auto loaded = load(bytes);
+    ASSERT_EQ(loaded, limit);
+
+    /// But it becomes "R\b" after assigning to SigningInput
+    Proto::SigningInput input;
+    input.set_gas_limit("R\b");
```
Comment (alejandro-isaza) [inline tests/TWEthereumSignerTests.cpp]: Protobuf uses strings to represent data, which I think is confusing. Use `set_gas_limit(data.data(), data.size())`

File: swift/Podfile [hunk 2]
```diff
@@ -4,6 +4,7 @@ target 'TrustWalletCore' do
   use_frameworks!
 
   pod 'SwiftProtobuf', '~> 1.2.0'
+  pod 'BigInt'
```
Comment (alejandro-isaza) [inline swift/Podfile]: Noo

Comment (hewigovens) [reply swift/Podfile]: It’s temporary to demonstrate the bug, will remove it

File: tests/TWEthereumSignerTests.cpp (hunk 1 above)
Comment (hewigovens) [reply tests/TWEthereumSignerTests.cpp]: Any idea about why this happens?

Comment (alejandro-isaza) [reply tests/TWEthereumSignerTests.cpp]: Why Protobuf uses string for data? It's a mystery. My guess is junior devs made it originally and then it was too hard to change.

Comment (hewigovens) [reply tests/TWEthereumSignerTests.cpp]: 😂, anything wrong from our swift serialization? This bug is critical

Comment (alejandro-isaza) [reply tests/TWEthereumSignerTests.cpp]: fixed :)

**Pull request merged**